src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContext.cs

```diff
@@ -16,6 +16,28 @@ public class TagHelperContext
         /// <summary>
         /// Instantiates a new <see cref="TagHelperContext"/>.
         /// </summary>
+        /// <param name="tagName">The HTML element's tag name.</param>
+        /// <param name="allAttributes">Every attribute associated with the current HTML element.</param>
+        /// <param name="items">Collection of items used to communicate with other <see cref="ITagHelper"/>s.</param>
+        /// <param name="uniqueId">The unique identifier for the source element this <see cref="TagHelperContext" />
+        /// applies to.</param>
+        public TagHelperContext(
```

Comment:
Both work fine. One just results in the `TagName` being `null` so a user can't determine the parsed `TagName`.